application.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import click
+
+
+@click.command()
+@click.argument('path', default='')
```

Comment:
Usually, CLI arguments will start with '--'

Reply:
Also, you can add if the argument is required or not using 'required' from 'Click'.